package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "logstar",
+  "version": "1.0.0",
+  "description": "Simple, opinionated, isomorphic logging.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dispatchme/logstar.git"
+  },
+  "license": "MIT",
+  "scripts": {
+    "lint": "./node_modules/.bin/eslint ./lib",
+    "prepublish": "rm -rf dist/** && ./node_modules/.bin/babel --ignore __tests__ lib --out-dir dist",
```

Comment:
These should work fine with just `eslint` and `babel` instead of `./node_modules/.bin/`, and that way they work on later node versions, which I don't think have `.bin` anymore. afaik, it will look in `./node_modules` for any commands you do in the scripts automatically.